.github/workflows/chrome-store-upload-dev.yaml

```diff
@@ -0,0 +1,31 @@
+name: Upload Dev To Store
+
+on: 
+  push:
+    branches:
+      - master
+      
+jobs:
+  build:
+    name: Upload-dev
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+        
+      - name: zip
+        uses: montudor/action-zip@v0.1.0
+        with:
+          args: zip -r ./vscd.zip ./
+        
+      - name: Upload to 
+        uses: trmcnvn/chrome-addon@v1
+        with:
+          # extension is only necessary when updating an existing addon,
+          # omitting it will create a new addon
+          extension: video-speed-controller-dev
```

Comment:
Sorry I seem to have missed it, where will this be uploaded to/how can we install it or instruct others to install?

Reply:
I would need to register a new extension on Chrome Store and then set it up here. The install process would be like any other extension once that's live.